pycoshark/mongomodels.py

```diff
@@ -779,7 +779,7 @@ class IssueEvent(Document):
     created_at = DateTimeField()
     status = StringField(max_length=50)
     author_id = ObjectIdField()
-    commit_id = ObjectIdField()
+    commit_id = StringField()
```

Comment:
Same as above

Reply:
I'm now storing the commit sha in commit id. I'm planing to revisit all the modules that has commit_id field in it after we finish vcsShark and update the actual commit module.

Reply:
Then it should be commit_sha not commit_id.

Reply:
done